conda-forge.yml

```diff
@@ -1,6 +1,8 @@
 bot:
   automerge: true
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
I have thought it over a bit and it is more complicated because the `RPATH` is not fixed after installing, so even if setting `CMAKE_INSTALL_RPATH` would have worked, the user would not have had a functional environment. I need to consult more on the matrix room to see what can be done